src/gui/src/KeySequence.cpp

```diff
@@ -191,7 +191,7 @@ QString KeySequence::keyToString(int key)
         {
             case Qt::LeftButton: return "1";
             case Qt::RightButton: return "2";
-            case Qt::MidButton: return "3";
+            case Qt::MiddleButton: return "3";
```

Comment:
Which is the minimum version of Qt that has this API?

Reply:
https://doc.qt.io/qt-5/qt.html#MouseButton-enum Qt 5.15 seems to have it. I'm not sure about older versions...